src/selectors/vaccine.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2021
+ */
+
+export const selectScannedSensors = ({ state }) => {
```

Comment:
Side note: I had trouble when using this - as I was passing in the whole state, and the reducer isn't called, so there isn't event a `vaccine` object in state. To get things working, I've added the vaccine reducer to the index.js and have used `vaccine` here rather than `state` 😃